recipes/scalene/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set package_name = "scalene" %}
+{% set version = "conda_2022_08_07" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://github.com/plasma-umass/scalene/archive/refs/heads/master.zip
+  url: https://github.com/plasma-umass/scalene/archive/refs/tags/conda_2022_08_07.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 5ee54e2aa4fb5e1ebe8b174004b9240f4b0a6fecd68722f5d2e84c768dc55026
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pip
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich
+    - cloudpickle
+    - numpy
+    - pynvml
+
+test:
+  imports:
+    - {{ package_name }}
+  source_files:
+    - test/testme.py
+  commands:
```

Would suggest adding this to validate dependencies. This can help with validating autoupdate PRs that Filipe mentioned above

```suggestion
  requires:
    - pip
  commands:
    - pip check
```